docker/templates/rollup-config.json

```diff
@@ -49,7 +49,7 @@
         "gas_price_diff": 50000
       },
       "chain_monitor": {
-        "enabled": false,
+        "enabled": true,
         "timeout": 3,
```

Comment:
Why contract need `bridge-history-api`, `rollup`, `chain-monitor` config

Reply:
This is mainly because we want to have a single command to generate all config files for scroll-sdk, so here the file https://github.com/scroll-tech/scroll-contracts/blob/feat-deployment-alt-gas-token/docker/scripts/gen-configs.sh will help to do so.